### PR TITLE
feat(plays): add Dice Simulator play to the project

### LINKED ISSUE #1342: [Add a Play]: Dice Simulator
### What's the Play about?

A dice simulator for RPGs such as DnD, where you decide the dice range, if you add any bonus, or if you have an advantage or disadvantage so that you can play more than one dice at once.

### What ReactJS concept will be used to create this play?

- Components
- useEffect
- useState

### Record

- [X] I have checked the existing [issues](https://github.com/reactplay/react-play/issues)
- [X] I have read the [Contributing Guidelines](https://github.com/reactplay/react-play/blob/main/CONTRIBUTING.md)
- [X] I want to work on this issue
- [x] I am in Hacktoberfest

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Roassaf**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

Add play Dice Simulator, a plataform where you can simulate dice so that you can play RPGs or other games

Fixes #1342

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

I ran this locally and checked if the play works

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

# Screenshots or example output

image

### CODE REVIEW

File: src/plays/dice-simulator/Readme.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+A plataform where you can simulate dice so that you can play RPGs or other games
+
+## Consideration
+
+Update all considerations(if any)
+
+## Resources
+
+Update external resources(if any)
```
Comment (priyankarpal) [inline src/plays/dice-simulator/Readme.md:27]: remove these lines if you don't want to use

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by priyankarpal** [comment]
> > hello @Roassaf would you like to update us?
> 
> Hi @priyankarpal, I will fix the readme this week

cool

File: src/plays/dice-simulator/Readme.md (hunk 1 above)
Comment (Roassaf) [reply src/plays/dice-simulator/Readme.md]: Updated in the new commit

**Review by joshi-kaushal — APPROVED** [review]

**Review by priyankarpal — APPROVED** [review]

**Comment by priyankarpal** [comment]
@Roassaf I would like to suggest that when you push your changes, you create a separate branch.

image

**Pull request merged**